invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -14,6 +14,8 @@
 from invenio_vocabularies.proxies import current_service as vocabulary_service
 from marshmallow import Schema, fields, missing, post_dump
 
+from ..dublincore.schema import DublinCoreSchema
+from ..schemas import CommmonSchema
```

Comment:
This import could be removed, and make `MARCXMLSchema(CommmonSchema):` to make use of inheritance. WDYT?

Reply:
If we go with inheritance I would rename it to `BaseSchema`.

Reply:
Hadn't though of inheritance, have now implemented both comments :)
GH CI tests in progress